### PR TITLE
added fixed issues

### PR DESCRIPTION
**Pull request by lennessyy**
Added fixed issues

### CODE REVIEW

File: modules/release-notes/pages/index.adoc [hunk 1]
```diff
@@ -67,6 +67,47 @@ TigerGraph Kubernetes Operator allows you to automate operations such as the cre
...
+|3.6.0
+|Calling `vertex.edgeAttribute()` function with `STRING COMPRESS` data type causes GPE crash
+|Unresolved
+
+|3.6.0
+|Interpreting a query that uses bitwise accumulators causes GPE dysfunction.
+|Unresolved
+
+|3.6.0
+|When vertices are automatically created during edge loading, the primary key of the of vertices created are not written to vertex attributes, even if the vertex definition has `primary_key_as_attribute` set to true.
```
Comment (arunramasami) [inline modules/release-notes/pages/index.adoc:90]: Let's remove "even if the vertex definition has `primary_key_as_attribute` set to true."

File: modules/release-notes/pages/index.adoc [hunk 2]
```diff
@@ -9,40 +10,102 @@ TigerGraph Server {page-component-version}.0 was released on June 15th, 2022.
...
+|3.6.0
+|Calling `vertex.edgeAttribute()` function with `STRING COMPRESS` data type causes GPE crash
+|Unresolved
+
+|3.6.0
+|Interpreting a query that uses bitwise accumulators causes GPE dysfunction.
+|Unresolved
+
+|3.6.0
+|When vertices are automatically created during edge loading, the primary key of the of vertices created are not written to vertex attributes.
```
Comment (a-m-thomas) [inline modules/release-notes/pages/index.adoc:90]: Extra "of"

File: modules/release-notes/pages/index.adoc [hunk 3]
```diff
@@ -9,40 +10,102 @@ TigerGraph Server {page-component-version}.0 was released on June 15th, 2022.
...
+|Unresolved
+
+
+|3.6.0
+|Running a query with `edgeAttribute()` or `neighborAttribute()` function when accumulator containing universal vertex types is referenced in the same statement cause GPE dysfunction.
```
Comment (a-m-thomas) [inline modules/release-notes/pages/index.adoc:95]: "may cause" or "causes" GPE dysfunction

File: modules/release-notes/pages/index.adoc [hunk 4]
```diff
@@ -9,40 +10,102 @@ TigerGraph Server {page-component-version}.0 was released on June 15th, 2022.
...
+|3.6.0
+|Calling the `setAttr()` function in an interpreted query causes GPE dysfunction.
+|Unresolved
+
+|3.6.0
+|High-frequency schema changes can cause GPE dysfunction in rare cases.
+|Unresolved
+|===
+
+For a list of known issues for GraphStudio and Admin portal, please see xref:gui:graphstudio:known-issues.adoc[Known issues for GraphStudio] and xref:gui:admin-portal:known-issues.adoc[Known issues for Admin Portal].
```
Comment (a-m-thomas) [inline modules/release-notes/pages/index.adoc:108]: Capitalize first instance of "Portal"

**Review by alexthomasTG — APPROVED** [review]

File: modules/release-notes/pages/index.adoc (hunk 1 above)
Comment (arunramasami) [inline modules/release-notes/pages/index.adoc]: Remove extra "of". the primary **key of the vertices** created **is** not written to vertex attributes

**Pull request merged**